src/textual/app.py

```diff
@@ -1489,7 +1489,7 @@ async def invoke_ready_callback() -> None:
                 finally:
                     self._mounted_event.set()
 
-                Reactive._initialize_object(self)
+                Reactive._initialize_reactable_object(self)
```

Comment:
as mentioned in the PR comments, I renamed methods in `Reactive` to be `_verb_reactive` or `_verb_reactable_object`. There is detailed explanation at the top of `reactive.py`.